nuxt.config.ts

```diff
@@ -22,7 +26,13 @@ export default defineNuxtConfig({
       app: 'docs',
     },
   },
-  routeRules: { ...redirects },
+  routeRules: {
+    ...redirects,
+    '/zk-stack/prividium': {
+      prerender: true,
+      ssr: true,
+    },
+  },
```

Comment:
I'm not sure of any other particular solution to this, so I guess we implement this for now if it works. Not too comfortable with putting something so specific but I'd rather not waste too much time on this. Does make me wonder if there's something in the `redirects` that's causing this issue somehow but I have no idea what.

Reply:
I removed the old redirects page [here](https://github.com/matter-labs/zksync-docs/pull/461), but it didn't change anything with this issue.